biolark/pom.xml

```diff
@@ -86,6 +86,7 @@
       <groupId>net.sf.json-lib</groupId>
       <artifactId>json-lib</artifactId>
       <classifier>jdk15</classifier>
+      <version>2.3</version>
```

Comment:
Ideally, the code should be updated to use the new `org.json` library instead of `json-lib`.